--- origsrc/skrooge/skgbaseguitest/CMakeLists.txt	2009-05-31 15:45:47.000000000 -0500
+++ src/skrooge/skgbaseguitest/CMakeLists.txt	2009-07-27 12:01:37.875004300 -0500
@@ -21,11 +21,11 @@ QT4_AUTOMOC(skgtestdateedit.cpp)
 QT4_AUTOMOC(skgtestmainpanel.cpp)
 QT4_AUTOMOC(skgtestwidgetcollection.cpp)
 
-TARGET_LINK_LIBRARIES(skgtestcombobox ${QT_QTCORE_LIBRARY} ${QT_QTTEST_LIBRARY} skgbasegui)
-TARGET_LINK_LIBRARIES(skgtestcalculatoredit ${QT_QTCORE_LIBRARY} ${QT_QTTEST_LIBRARY} skgbasegui)
-TARGET_LINK_LIBRARIES(skgtestdateedit ${QT_QTCORE_LIBRARY} ${QT_QTTEST_LIBRARY} skgbasegui)
-TARGET_LINK_LIBRARIES(skgtestmainpanel ${QT_QTCORE_LIBRARY} ${QT_QTTEST_LIBRARY} skgbasegui skgbasemodeler)
-TARGET_LINK_LIBRARIES(skgtestwidgetcollection ${QT_QTCORE_LIBRARY} ${QT_QTTEST_LIBRARY} skgbasegui)
+TARGET_LINK_LIBRARIES(skgtestcombobox ${KDE4_KDEUI_LIBS} ${QT_QTCORE_LIBRARY} ${QT_QTTEST_LIBRARY} skgbasegui)
+TARGET_LINK_LIBRARIES(skgtestcalculatoredit ${KDE4_KDECORE_LIBS} ${QT_QTGUI_LIBRARY} ${QT_QTTEST_LIBRARY} skgbasegui)
+TARGET_LINK_LIBRARIES(skgtestdateedit ${KDE4_KDECORE_LIBS} ${QT_QTGUI_LIBRARY} ${QT_QTTEST_LIBRARY} skgbasegui)
+TARGET_LINK_LIBRARIES(skgtestmainpanel ${KDE4_KDECORE_LIBS} ${QT_QTGUI_LIBRARY} ${QT_QTTEST_LIBRARY} skgbasegui skgbasemodeler)
+TARGET_LINK_LIBRARIES(skgtestwidgetcollection ${KDE4_KDECORE_LIBS} ${QT_QTGUI_LIBRARY} ${QT_QTTEST_LIBRARY} skgbasegui)
 
 #Add test
 ENABLE_TESTING()
